tests/run-make/extern-fn-explicit-align/test.rs

```diff
@@ -34,6 +34,13 @@ pub struct LowerAlign {
     pub b: u64,
 }
 
+#[derive(Copy, Clone)]
+#[repr(C)]
+#[repr(packed)]
+pub struct Packed {
+    pub a: u128
+}
```

Comment:
Any bets on which of our targets has the wrong ABI for this?

We've seen weird special cases from -msvc and -gnu already, so I think it's time for Apple to get in the ring.